### PR TITLE
andysprout3

### PR DESCRIPTION
**Pull request by AndyAn7**
made deletions of fortawesome devs to clear up package-lock in client; ran npm audit fix --force to clean up package-lock before posting PR..thx

### CODE REVIEW

**Comment by AndyAn7** [comment]
Header/Footer initial addition styling: done.

**Comment by AndyAn7** [comment]
sorry for all the commits..it was an all nighter

**Comment by frye** [comment]
A commit for a change that you can test and validate, or a bug fix is the right thing to do. That way you can always go back in time if things go south. Less commits and you will have more to catch up if you have to restart.

File: client/package.json [hunk 1]
```diff
@@ -37,5 +40,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "sass": "^1.51.0"
```
Comment (frye) [inline client/package.json:45]: Why did you move sass to dev dependencies? It might be right place for it I just want to be sure.

File: client/src/components/Home/index.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+import Header from '../Header';
+import Footer from '../Footer';
+import bg from '../../assets/images/projectPhoto.jpg'
 
 const Home = () => {
     return (
         <>
-            <p>Hello World!</p>
+            <Header />
+            <img src={bg} alt='' style={{ minHeight: '100%', minWidth: '100%', borderRadius: '20px', opacity: '.2' }}>
```
Comment (frye) [inline client/src/components/Home/index.js:9]: Is this the background image? If yes then I'd consider to put it to the top level component. Also to be consistent across files the styling should go to the style sheet.

**Comment by frye** [comment]
@AndyAn7 overall looks great! I left few comments to clarify things. As this is how PRs are normally reviewed. You know your changes best so there may be a reason you did things the way you did. Do not expect me to be right necessarily.

**Comment by AndyAn7** [comment]
K thank you!

**Review by frye — APPROVED** [review]

File: client/src/components/Home/index.js (hunk 2 above)
Comment (AndyAn7) [reply client/src/components/Home/index.js:9]: Alright, I just saw this, thank you..

**Comment by AndyAn7** [comment]
Andrew M. An
(253) 414-5536

Begin forwarded message:

> From: Framer Team ***@***.***>
> Date: May 31, 2022 at 8:07:29 AM PDT
> To: Andrew An ***@***.***>
> Subject: Multiple Pages + Site Spotlight
> Reply-To: ***@***.***
> 
> ﻿
> 
> Hey Andrew, Framer Sites launched last week! Everyone can now enjoy breakpoints, effects, links, styles, and the cms to design and publish incredible websites. We even made it to the top spot on Product Hunt. Thank you for the support!
> 
> If you haven’t had a chance to play around, we highly recommend the new tutorial. You’ll have your first site up and running in about 10 minutes.
> 
> Page Types
> 
> You might have seen that every new Framer project now shows you three new page types in the sidebar.
> 
>  
> Web pages have a unique URL and user interface for setting up breakpoints and adding effects. Web pages can be linked and published.
> 
> CMS Pages are templates that pull data from the CMS and generate web pages for you.
> 
> Canvas Pages are for ideating and prototyping. You can insert prototypes on web pages and present them next to feedback forms.
> 
> Here’s a quick guide if you want to learn more.
> 
> Site Spotlight
> 
> Aaron Abentheuer and the team at Playground launched a new site for WorkLens, an intelligent productivity app for people who take photos to get things done. It’s full of cool hover and scroll effects and works flawlessly across all screen sizes.
> 
> “The site was built with the new Framer, and I’ll say it again: for me it’s hands down the future of building websites. At playgr_und we don’t even go to Figma anymore to design sites, we just start building straight away in @framer.” — Aaron on Twitter
> 
> 
> Looking for inspiration? We’re adding new sites to the gallery every week.
> 
> That’s it for this week! Thanks for reading, and keep up the good work.
> 
> Cheers,
> ✌️
> 
> Unsubscribe
>
